google/cloud/spanner/samples/postgresql_samples.cc

```diff
@@ -260,13 +726,25 @@ CommandType HelpCommand(std::map<std::string, CommandType>& commands) {
 int RunOneCommand(std::vector<std::string> argv,
                   std::string const& extra_help) {
   std::map<std::string, CommandType> commands = {
+      {"create-clients", Command(samples::CreateClients)},
       {"create-database", Command(samples::CreateDatabase)},
       {"create-tables", Command(samples::CreateTables)},
       {"add-column", Command(samples::AddColumn)},
-      {"create-storing-index", Command(samples::CreateStoringIndex)},
       {"insert-data", Command(samples::InsertData)},
       {"query-with-parameter", Command(samples::QueryWithParameter)},
       {"dml-getting-started-update", Command(samples::DmlGettingStartedUpdate)},
+      {"batch-dml", Command(samples::BatchDml)},
+      {"drop-tables", Command(samples::DropTables)},
```

Comment:
Your call, but if this is internal-only, maybe we can omit it here.

Reply:
The person exercising individual samples from the command line will still want to call `CreateTables()` and `DropTables()`, but I agree that maintaining state between samples makes that process error prone.

I'm going to move them out of the `samples` namespace, and give them non-tag-like banners.